data-analysis-requirements.txt

```diff
@@ -0,0 +1,17 @@
+healpy
```

Comment:
so you are not using it anymore, can be removed?

Reply:
correct, done